dags/iop/parser.py

```diff
@@ -33,5 +38,22 @@ def _get_dois(self, article: ET.Element):
         dois = node.text
         if dois:
             self.logger.msg("Parsing dois for article", dois=dois)
+            self.dois = dois
             return [dois]
-        return None
+        return
+
+    def _get_journal_doctype(self, article: ET.Element):
+        node = article.find(".")
+        try:
+            value = node.get("article-type")
+            return self.article_type_mapping[value]
+        except KeyError:
+            self.logger.error(
+                "Unmapped article type",
+                doi=self.dois,
+                article_type=value,
+                article_type_mapping=self.article_type_mapping,
```

Comment:
I added it here because of the type, which was found in the XML might not be in `article_type_mapping`

Reply:
yes, yes, I get it but the question is if you really want to have that in all the logs? I imagine for example the whole batch of the files that for some reason have an incorrect `article_type` and in every log message, you have this mapping (which is imo redundant, what if it will grow?) and you have to unfold the message on kibana. But it's just a detail.

Reply:
Understand, I just wanted to give the value which is received from xml and then just show it: LOOK IS NOT IN THE DICT, HERE YOU ARE CHECK IT. but if it gives more damage than value, I will remove it